tools/OutlineService/OutlineService/App.config

```diff
@@ -1,19 +1,4 @@
-﻿<!--
-  Copyright 2018 The Outline Authors
-
-  Licensed under the Apache License, Version 2.0 (the "License");
-  you may not use this file except in compliance with the License.
-  You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-  Unless required by applicable law or agreed to in writing, software
-  distributed under the License is distributed on an "AS IS" BASIS,
-  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  See the License for the specific language governing permissions and
-  limitations under the License.
--->
-<?xml version="1.0" encoding="utf-8" ?>
+﻿<?xml version="1.0" encoding="utf-8" ?>

```

Comment:
Is this change intentional?

Reply:
Yes, this was necessary to get Visual Studio to open the project (`<?xml>` must be the first element in the doc). I couldn't find a simple yes/no on whether copyright headers are required in auto-generated files like this; since we haven't been doing it for Xcode-generated files, I was hoping to at least be consistent (and make the files smaller).